### PR TITLE
Readme and Install files changed

### PR DESCRIPTION
**Pull request by iamflorencejay**
It is obvious

### CODE REVIEW

**Comment by iamflorencejay** [comment]
@s9260 im done what you've ask. 😆

**Review by s9260 — CHANGES_REQUESTED** [review]
Change L45 (readme) to:
```This project is licensed under the GNU General Public License Version 3.0. For the full license, see [LICENSE](LICENSE).```
Other than that, perfect except for screenshots.
Build off of bf826716fe3813b895f289b452862e80c9c17a34 (indev) to get closest to current production, I'm changing Views -> LBC so I stop abusing the views API endpoint so much.

**Comment by iamflorencejay** [comment]
@s9260 done

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Contributions to this project are welcome, encouraged, and compensated. For more details, see [CONTRIBUTING](https://lbry.tech/contribute).
+
+> NOTE:
+> Currently, the developer dosen't accept commits yet, because it is in development. You can see it [here.](https://github.com/lbryio/odysee-roku/issues/7)
+
+## License
+This project is licensed under the GNU General Public License Version 3.0. For the full license, see [LICENSE](LICENSE).
+
+## Security
+We take security seriously. Please contact security@lbry.com regarding any security issues. Our PGP key is [here](https://keybase.io/lbry/key.asc) if you need it. All releases are signed by [Sean Yesmunt](https://keybase.io/seanyesmunt/key.asc).
```
Comment (s9260) [inline README.md:51]:
```suggestion
We take security seriously. Please contact security@lbry.com regarding any security issues. Our PGP key is [here](https://keybase.io/lbry/key.asc) if you need it.
```

Comment (s9260) [inline README.md:51]:
```suggestion
We take security seriously. Please contact security@lbry.com regarding any security issues. Our PGP key is [here](https://keybase.io/lbry/key.asc) if you need it.
```

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+2. Make a note of the on-screen IP address or check your roku's IP address on your router.
+3. Select I Agree on the SDK License page.
+4. Create a password.                                             
+5. Enter the previously-noted IP address into your browser.       
+6. Click on Upload.                                               
+7. Select the ZIP file of the channel you want to install.
+8. Click on Install.
+
+## Developer Instructions                                         
+If you want to debug, clone this repository and go to roku-main-channel folder. You can see the core files such as components, images, etc.
```
Comment (s9260) [inline INSTALL.md:16]:
```suggestion
If you want to debug, clone this repository and go to the MVP-Roku folder. You can see the core files such as components, images, etc.
```

**Pull request merged**